packages/starksheet-webapp/src/contexts/OnsheetContext.tsx

```diff
@@ -73,21 +69,17 @@ export const OnsheetContextProvider = ({
     () =>
       Promise.all([
         contract.getSheetDefaultRendererAddress(),
-        contract.getSheetClassHash(),
-        contract.getProxyClassHash(),
         contract.getSheetPrice(),
       ])
         .then(async (response) => {
-          const [renderer, sheetClassHash, proxyClassHash, sheetPrice] =
+          const [renderer, sheetPrice] =
             response;
           return {
             address,
             defaultRenderer: renderer,
-            sheetClassHash,
-            proxyClassHash,
             sheets: [],
             sheetPrice:
-              sheetPrice.div(number.toBN(10).pow(number.toBN(9))).toNumber() /
+              sheetPrice.div(new BN(10).pow(new BN(9))).toNumber() /
```

Comment:
I did this "hack" to go from a uint with 18 decimals price (in wei) to a float representation in eth. there is probably a better way to do ti